src/main/java/com/boardgo/domain/chatting/service/ChatMessageQueryServiceV1.java

```diff
@@ -16,7 +26,24 @@ public class ChatMessageQueryServiceV1 implements ChatMessageQueryUseCase {
 
     @Override
     public List<LatestMessageResponse> getLast(List<Long> roomIdList) {
-        return chatMapper.toLatestMessageResponse(
-                chatRepository.findLatestMessagesByRoomIds(roomIdList));
+        List<LatestMessageProjection> result =
+                chatRepository.findLatestMessagesByRoomIds(roomIdList);
+        for (LatestMessageProjection latestMessageProjection : result) {
+            log.info("latestMessageProjection.roomId = {}", latestMessageProjection.roomId());
+            log.info(
+                    "latestMessageProjection.latestMessage.Content = {}",
+                    latestMessageProjection.latestMessage().getContent());
```

Comment:
이 부분 디버깅하는 이유가 궁금해요
그런데 로그를 위해 for문으로 출력하게 되면 O(n) 시간이 발생해서 로그갯수를 줄이거나 로그레벨을 낮추는건 어떨까 해요!
```suggestion
            log.info("roomId={}, latestMessage.Content={}", 
             latestMessageProjection.roomId(), 
             latestMessageProjection.latestMessage().getContent());
```

Reply:
아 이거 어차피 디버깅하고 모두 지울거라서 괜찮을 거 같습니다! 
최대한 한 번의 로깅 업데이트로 모든 부분을 확인하려고 하는거라 .. ㅠㅠ